packages/number-input/src/NumberInput/NumberInput.spec.tsx

```diff
@@ -8,7 +8,6 @@ import { NumberInput } from '.';
 
 const label = 'This is the label text';
 const description = 'This is the description text';
-const errorMessage = 'error message';
 const arrowTestId = {
   up: 'lg-number_input-increment_button',
```

Comment:
Could we make these consts?

Reply:
assuming the question is about the arrow test ids, I think I'll leave this as-is for now since it requires a further refactor of the Arrows/Arrow components. Maybe can get a refactor when adding test harnesses to this package